### ISSUE TITLE
Activity types aren't respected in Send Activity / LG

### ISSUE THREAD

**Issue by darrenj** [microsoft/BotFramework-Composer#1508]
## Describe the bug
When providing an Activity structure in a LG response which changes the `Type` it's ignored and always sent as a `message`

![image](https://user-images.githubusercontent.com/33519520/68234585-8bc88c00-ffcf-11e9-8c12-9a7c4f34966c.png)

Results in the following which is incorrect:

![image](https://user-images.githubusercontent.com/33519520/68234720-d518db80-ffcf-11e9-8d4f-d329d628e4e8.png)

![image](https://user-images.githubusercontent.com/33519520/68234727-d944f900-ffcf-11e9-9dae-3c4c4f0a52f8.png)

## Expected behavior
- Able to control the Activity `type`, for example an `event` or `handoff`

**Comment by hibrenda** [comment]
![image](https://user-images.githubusercontent.com/43408889/68278420-3910d200-00ac-11ea-87f1-c55173908c10.png)

[Activity
         Name = eventActivity
         Value = eventValue
         Type = event
 ]

**Comment by darrenj** [comment]
Whilst that helps for the event type - any other type is ignored as detailed above. e.g. if you set type to "handoff" it is ignored silently.

**Comment by hibrenda** [comment]
> Whilst that helps for the event type - any other type is ignored as detailed above. e.g. if you set type to "handoff" it is ignored silently.

Yes. "handoff" is not supported . This is an known issue and tracked in SDK .
